modules/dr-restoring-cluster-state.adoc

```diff
@@ -3,6 +3,12 @@
 // * disaster_recovery/scenario-2-restoring-cluster-state.adoc
 // * post_installation_configuration/cluster-tasks.adoc
 
+// Contributors: Some changes for the "etcd" restore procedure are only valid for 4.14+. 
```

Comment:
```suggestion
// Contributors: Some changes for the `etcd` restore procedure are only valid for 4.14+. 
```

Reply:
@tmalove Just one minor nit! Use as you see fit! :+1:

Reply:
Thanks @GroceryBoyJr ...done!